packages/dma-contracts/test/integration/morphoblue/200_morpho-borrow.ts

```diff
@@ -315,8 +321,57 @@ describe('Borrow Operations | MorphoBlue | Integration', async () => {
         testSystem,
         market,
         user,
-        borrowAmount,
+        diminishedRepayAmount,
+        diminishedWithdrawAmount,
+        false,
+      )
+
+      expect(successPaybackWithdraw).to.be.true
+
+      if (market.collateralToken === 'WETH') {
+        expect(userEthBalanceAfter).to.be.equal(
+          userEthBalanceBefore
+            .add(diminishedWithdrawAmount)
+            .sub(receipt.gasUsed.mul(receipt.effectiveGasPrice)),
+        )
+      } else {
+        expect(collateralBalanceAfter).to.be.equal(
+          collateralBalanceBefore.add(diminishedWithdrawAmount),
+        )
+      }
+
+      expect(loanTokenBalanceAfter).to.be.equal(loanTokenBalanceBefore.sub(diminishedRepayAmount))
+    }
+  })
+
+  it('should be able to payback all + withdraw (with loan token)', async () => {
+    for (const market of morphoSystem.marketsInfo) {
+      const {
+        success: successDepositBorrow,
+        collateralAmount,
+        borrowAmount: repayAmount,
+      } = await opMorphoBlueDepositBorrow(testSystem, market, user)
+
+      expect(successDepositBorrow).to.be.true
+
+      const augmentedRepayAmount = repayAmount.add(repayAmount.div(10))
```

Comment:
can you try increasing timestamp by rpc method `evm_increaseTime` so we can verify that the debt actually accrues